apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -398,6 +412,23 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
                   signatory={workingCase.judge?.name}
                   signingDate={workingCase.rulingDate}
                 />
+                {(user?.role === UserRole.JUDGE ||
+                  user?.role === UserRole.REGISTRAR) && (
```

Comment:
Should not only the judge be able to sign the ruling?
```suggestion
                {user?.role === UserRole.JUDGE && (
```